catalog-info.yaml

```diff
@@ -22,9 +22,14 @@ spec:
     spec:
       repository: elastic/package-registry
       pipeline_file: ".buildkite/pipeline.yml"
+      branch_configuration: "main v1.* v2.*"
```

Comment:
My only concern is that every time we release a new major we should update this `branch_configuration`. 
We could avoid this by saying `v*`?

Reply:
Yes, that would allow every tag pushed. The only drawback is that someone pushes a branch starting with `v*` (just an example vulnerability_package), that would also trigger a build in Buildkite.

Maybe, for now we can try that other glob pattern that is more "open". And if needed, we can restrict (add more globs with v1.*, v2.*) later. Agreed ? @alexsapran

Reply:
+++